src/main/java/ChatBot.java

```diff
@@ -0,0 +1,154 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class ChatBot {
+    private String chatbotName;
+    private List<Task> tasks;
+    private Scanner in;
+
+    public ChatBot(String chatbotName) {
+        this.chatbotName = chatbotName;
+        this.tasks = new ArrayList<>();
+        this.in = new Scanner(System.in);
+    }
+
+    public void startChat() {
```

Comment:
Good method name that clearly tells that the method is to start the chat as well as for the printWelcomeMessage in the next line.